index.js

```diff
@@ -42,7 +42,20 @@ const getRandomChar = stack => {
 };
 
 const produce = (number, callback) => {
-  const result = [];
-  for (let index = 0; index < number; index++) result.push(callback(index));
-  return result.join("");
+  let result = "";
+  for (let index = 0; index < number; index++) result += callback(index);

```

Comment:
Not super important, just a suggestion in case if you want to save a few bytes. Could be transformed into something like:
```suggestion
  for (var index = 0, result = ''; index < number; index++) result += callback(index);
```

But will only work with `var` since `let` scopes the variable differently.